runtime/amd64.S

```diff
@@ -1378,7 +1395,7 @@ G(caml_system.frametable):
         .globl  G(caml_negf_mask)
         .align  SIXTEEN_ALIGN
 G(caml_negf_mask):
-        .quad   0x8000000000000000, 0
+        .quad   0x8000000000000000
```

Comment:
This change does not seem to be related to the intent of this PR.

Reply:
Yeah, mistake on my part. Previous version had me tweaking something here for alignment for some reason. I'll fix that.